docs/cluster_spec.md

```diff
@@ -0,0 +1,46 @@
+# Description of Keys in `config` and `cluster.spec`
+
+This list is not complete, but aims to document any keys that are less than self-explanatory.
+
+## spec
+
+### adminAccess
+
+This array configures the CIDRs that are able to ssh into nodes. On AWS this is manifested as inbound security group rules on the `nodes` and `master` security groups.
+
+Use this key to restrict cluster access to an office ip address range, for example.
+
+```yaml
+spec:
+  adminAccess:
+    - 12.34.56.78/32
+```
```

Comment:
adminAccess is documented on the arguments page https://github.com/kubernetes/kops/blob/master/docs/arguments.md